src/Storefront/Resources/views/storefront/layout/footer/footer-minimal.html.twig

```diff
@@ -2,8 +2,6 @@
 
 {% block layout_footer_inner_container %}
     <footer class="footer-minimal">
-        {% block layout_footer_bottom %}
-            {{ parent() }}
-        {% endblock %}
+        {{ block("layout_footer_bottom") }}
```

Comment:
Sure we can use this with multi inheritance? We do not have `sw_block` yet.

Reply:
Afaik this is only a problem, if the block is used from another template, i.e. `{{ block('foo', '@Namespace/path/to/template.html.twig') }}`, as the inheritance of  `@Namespace/path/to/template.html.twig` is not properly resolved. But maybe @JoshuaBehrens can elaborate on this, as I am not 100% sure.

Reply:
as @aragon999 said the render block function will work easily with block without a second parameter. When we have a 2nd parameter the inheritance check is missing. So this is fine.

sw_block is about to be merged so maybe this can be part of the new tool @shyim is working on?